src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn

```diff
@@ -52,29 +54,105 @@
       <bpmn:outgoing>SequenceFlow_0v6jrro</bpmn:outgoing>
     </bpmn:exclusiveGateway>
     <bpmn:sequenceFlow id="SequenceFlow_0i3vch4" sourceRef="ExclusiveGateway_0yphhsk" targetRef="EndEvent_COMP_SERVICE_DRAFT" />
-    <bpmn:sequenceFlow id="SequenceFlow_0ds01g1" sourceRef="ServiceTask_UpdateTeamByStageAndTexts_QA" targetRef="EndEvent_COMP_SERVICE_DRAFT" />
-    <bpmn:sequenceFlow id="SequenceFlow_1v0uel3" sourceRef="ExclusiveGateway_0yphhsk" targetRef="ServiceTask_UpdateTeamByStageAndTexts_QA">
+    <bpmn:sequenceFlow id="SequenceFlow_0ds01g1" sourceRef="Service_UpdateTeamByStageAndTexts_QA" targetRef="EndEvent_COMP_SERVICE_DRAFT" />
+    <bpmn:sequenceFlow id="SequenceFlow_1v0uel3" sourceRef="ExclusiveGateway_0yphhsk" targetRef="Service_UpdateTeamByStageAndTexts_QA">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctDraftResult == "QA"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="SequenceFlow_0wewq10" sourceRef="ExclusiveGateway_0yphhsk" targetRef="ServiceTask_UpdateTeamByStageAndTexts_Send">
+    <bpmn:sequenceFlow id="SequenceFlow_0wewq10" sourceRef="ExclusiveGateway_0yphhsk" targetRef="Service_UpdateTeamByStageAndTexts_Send">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctDraftResult == "Send"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:serviceTask id="ServiceTask_UpdateTeamByStageAndTexts_Resp_Escalate" name="Update Team for Service Response Escalate" camunda:expression="${bpmnService.updateTeamByStageAndTexts(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey,&#34;COMP_SERVICE_RESP_ESCALATE&#34;,&#34;QueueTeamUUID&#34;,&#34;QueueTeamName&#34;,&#34;Stage&#34;)}">
-      <bpmn:incoming>SequenceFlow_03k2r1l</bpmn:incoming>
+    <bpmn:serviceTask id="Service_UpdateTeamByStageAndTexts_Resp_Escalate" name="Update Team for Service Escalate" camunda:expression="${bpmnService.updateTeamByStageAndTexts(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey,&#34;COMP_SERVICE_ESCALATE&#34;,&#34;QueueTeamUUID&#34;,&#34;QueueTeamName&#34;,&#34;Stage&#34;)}">
```

Comment:
Should this still be called ..._Resp_Escalate as we have renamed elsewhere to Service Escalate?

Reply:
Fixed, thank you.